.circleci/config.yml

```diff
@@ -71,7 +71,7 @@ jobs:
                 command: |
                     export NVM_DIR="/opt/circleci/.nvm"
                     [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
-                    nvm use 10
+                    nvm install v14.17.5 --default
```

Comment:
Q: Should this be pinned to a specific version? Or wouldn't pinning generically to v14 use the latest (security) release in that channel? I guess my concern is that we'd have to constantly bump this if new versions of v14 are added for future security releases, etc.

Reply:
I think you're right – updated.